guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<p>can you change the intro line to&nbsp;from <strong>existing </strong>mosaic datasets</p><p>Adds&nbsp;<code>md_to_upload</code>&nbsp;parameter to allow publishing&nbsp;<code>ImageryLayers</code>&nbsp;from <strong>existing </strong>mosaic datasets in&nbsp;<code>ArcGIS Online</code>&nbsp;to:</p><p> </p><p>Also this enhancement does not look new - Adds support for path to local rasters as&nbsp;<code>input_rasters</code>&nbsp;argument</p><p>Could you share the PR or issue that you referred to? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Hi @rhea-j</p><p><a href="https://github.com/ArcGIS/geosaurus/pull/7057" target="_blank">PR #7057</a> - it added a sentence to the parameter table so I assumed this was new functionality. Let me know if it should be removed.</p>

Reply:
I revised note to read. Let me know if it should be removed:
_* Adds documentation to highlight support for local rasters as `input_rasters` argument_